docs/changes/2732.bugfix.rst

```diff
@@ -1 +0,0 @@
-Fix units (absence of) for skewness and excess kurtosis in muon analysis
```

Comment:
Why do you do this, this is not related to your changes, please do not touch, otherwise you will have merge conflicts in the future or risk removing an entry from the changelog